crawl-ref/docs/options_guide.txt

```diff
@@ -2068,6 +2069,16 @@ tile_font_lbl_size  = 14
 tile_font_ft_light = false
         Selects 'light' font hinting.
 
+glyph_mode_font = monospace
+        Font used to render the dungeon in the glyph mode for WebTiles. If set
+        to monospace (default) or the font is not available, browser's default
```

Comment:
```suggestion
        to monospace (default) or the font is not available, the browser's default
```

Reply:
Thanks for pointing this out. I have made the suggested change in my local repository, and I'll hold off updating the PR until code reviews come in.